pages/participantinfo.tsx

```diff
@@ -0,0 +1,516 @@
+    Textarea,
+    Heading,
+    UnorderedList,
+    ListItem,
+    OrderedList,
+    Radio,
+    RadioGroup,
+} from "@chakra-ui/react";
+import { ArrowBackIcon, CloseIcon } from "@chakra-ui/icons";
+import { useState } from "react";
+import { useSession } from "next-auth/client";
+import Wrapper from "@components/SDCWrapper";
+
+const BLUE = "#0C53A0";
+const RADIO_YES = "yes";
+const RADIO_NO = "no";
+
+const FormButton = (props) => {
+    return (
+        <Button
+            bg={BLUE}
+            color={"white"}
+            fontWeight="400"
+            onClick={props.onClick}
+            my={8}
+            px={12}
+        >
+            {props.children}
+        </Button>
+    );
+};
+
+const FormPage = (props) => {
+    return <Stack spacing={8}>{props.children}</Stack>;
+};
+
+/**
+ * This is the page that a user will use to enter the participants personal information
+ * onto the SDC platform as a parent of volunteer
+ */
+export default function ParticipantInfo(): JSX.Element {
+    const [session, loading] = useSession();
+    const [progressBar, setProgressBar] = useState(Number);
+    const [pageNum, setPageNum] = useState(0);
+    const [isOnMedication, setIsOnMedication] = useState(RADIO_NO);
+    const [hasAllergies, setHasAllergies] = useState(RADIO_NO);
+
+    const medicationDetails =
+        isOnMedication === RADIO_YES ? (
+            <Box>
+                <FormControl id="details">
+                    <FormLabel>
+                        Please provide any details if necessary{" "}
+                    </FormLabel>
+                    <Input placeholder="Details" />
+                </FormControl>
+            </Box>
+        ) : null;
+
+    const allergyDetails =
+        hasAllergies === RADIO_YES ? (
+            <Box>
+                <FormControl id="details">
+                    <FormLabel>
+                        Please provide any details if necessary{" "}
+                    </FormLabel>
+                    <Input placeholder="Details" />
+                </FormControl>
+            </Box>
+        ) : null;
+
+    const formPageHeaders = [
+        "Participant Information",
+        "Participant Information",
+        "Parent Guardian Information",
+        "Participant Emergency Form",
+        "Participant Health Form",
+        "Confirm Participants",
+        "Proof of Income",
+        "How did you hear about us?",
+    ];
+
+    const formPages = [
+        <FormPage>
+            <Box maxW="55rem">
+                <Text noOfLines={2} fontSize="16px" fontWeight="200">
+                    Please provide information on the participant that is being
+                    registered in the program. An opportunity to add information
+                    of additional participants you would like to register will
+                    be provided afterwards.
+                </Text>
+            </Box>
+            <FormLabel>
+                {" "}
+                Emergency Contact Name
```

Comment:
I feel like this should be asking for the participant name instead of emergency contact, pinging designers for confirmation

Reply:
Update: please change this to Participant Name